### PR TITLE
Remove NullnessRawnessChecker

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1128,10 +1132,10 @@ Renamed method annotations used by the Nullness Checker:
 Removed annotations used by the Nullness Checker:
   @AssertParametersNonNull
 Renamed type annotations used by the Initialization Checker:
-  @NonRaw => @Initialized
```
Comment (smillst) [inline changelog.txt]: The changes in the file should be reverted, except for the added entry at the top.

File: checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java [hunk 2]
```diff
@@ -72,10 +69,10 @@
                 Flow extends CFAbstractAnalysis<Value, Store, Transfer>>
         extends GenericAnnotatedTypeFactory<Value, Store, Transfer, Flow> {
 
-    /** {@link UnknownInitialization} or {@link Raw}. */
+    /** {@link UnknownInitialization}. */
     protected final AnnotationMirror UNCLASSIFIED;
```
Comment (smillst) [inline checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java]: Should we go ahead and rename all of these fields?  For example, this one could be` UNKNOWN_INITIALIZATION`.

File: checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java [hunk 3]
```diff
@@ -651,13 +605,13 @@ private void computeFieldAccessType(
                     isInitializedForFrame(receiverType, fieldDeclarationType);
             if (isInitializedForFrame) {
                 // The receiver is initialized for this frame.
-                // Change the type of the field to @UnknownInitialization or @Raw so that
+                // Change the type of the field to @UnknownInitialization or so that
```
Comment (smillst) [inline checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java]: remove "or"

File: checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java [hunk 4]
```diff
@@ -651,13 +605,13 @@ private void computeFieldAccessType(
...
                 // anything can be assigned to this field.
                 type.replaceAnnotation(UNCLASSIFIED);
             } else if (computingAnnotatedTypeMirrorOfLHS) {
                 // The receiver is not initialized for this frame, but the type of a lhs is being
                 // computed.
-                // Change the type of the field to @UnknownInitialization or @Raw so that
+                // Change the type of the field to @UnknownInitialization or so that
```
Comment (smillst) [inline checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java]: remove "or"

File: framework/src/main/java/org/checkerframework/framework/type/SubtypeVisitHistory.java [hunk 5]
```diff
@@ -8,8 +8,7 @@
 import org.checkerframework.javacutil.Pair;
 
 /**
- * THIS CLASS IS DESIGNED FOR USE WITH DefaultTypeHierarchy, DefaultRawnessComparer, and
- * StructuralEqualityComparer ONLY.
+ * THIS CLASS IS DESIGNED FOR USE WITH DefaultTypeHierarchy and StructuralEqualityComparer ONLY.
```
Comment (smillst) [inline framework/src/main/java/org/checkerframework/framework/type/SubtypeVisitHistory.java]: This should be reverted.   (Rawness here is referring to a generic type without a type argument.)

File: framework/tests/all-systems/InferTypeArgs2.java [hunk 6]
```diff
@@ -1,8 +1,7 @@
 class InferTypeArgs2<V extends InferTypeArgs2<V>> {}
 
 // having a concrete extension of InferTypeArgs2 is the key difference between this and
-// InferTypeArgs1.java.  In this case we end up comparing CFValue with V extends InferTypeArgs2<V>
```
Comment (smillst) [inline framework/tests/all-systems/InferTypeArgs2.java]: This should be reverted. (Rawness here is referring to a generic type without a type argument.)

**Comment by mernst** [comment]
@smillst Thanks for catching some things I had overlooked.
@wmdietl Do you want to take a look, or not?

File: checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java [hunk 7]
```diff
@@ -486,12 +452,8 @@ protected void setSelfTypeInInitializationCode(
...
                             .isEmpty()) {
                 if (classType.isFinal()) {
-                    annotation = COMMITTED;
-                } else if (useFbc) {
-                    annotation = createFreeAnnotation(classType);
-                } else {
-                    annotation = createUnclassifiedAnnotation(classType);
-                }
+                    annotation = INITIALIZED;
+                } else annotation = createFreeAnnotation(classType);
```
Comment (wmdietl) [inline checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java]: Nit: for symmetry with the `then` branch I would put this statement also into a block.

File: checker/src/main/java/org/checkerframework/checker/nullness/NullnessChecker.java [hunk 8]
```diff
@@ -3,7 +3,8 @@
 /** A concrete instantiation of {@link AbstractNullnessChecker} using freedom-before-commitment. */
 public class NullnessChecker extends AbstractNullnessChecker {
```
Comment (wmdietl) [inline checker/src/main/java/org/checkerframework/checker/nullness/NullnessChecker.java]: This class doesn't have a purpose any more. We could remove it and rename `AbstractNullnessChecker` to `NullnessChecker`.

**Comment by mernst** [comment]
@wmdietl Thanks for the useful comments.

**Pull request merged**